### ISSUE TITLE
How to have `global sys.version` to use Python 3 by default?

### ISSUE THREAD

**Issue by alwc** [davidhalter/jedi-vim#819]
### Issue

I followed [this setup](https://github.com/zchee/deoplete-jedi/wiki/Setting-up-Python-for-Neovim) to create two `pyenv`: `neovim2` and `neovim3`. jedi-vim works fine if I ran `pyenv activate neovim3` on prompt before starting vim. However, without activating the `neovim3` environment, it seems to be running python2 by default.

Is it possible to use Python3 by default? Thanks.

### Steps to reproduce

```
if has("unix")
    if has('mac')
        let g:python_host_prog = '/Users/alexlee/.pyenv/versions/neovim2/bin/python'
        let g:python3_host_prog = '/Users/alexlee/.pyenv/versions/neovim3/bin/python'
        let g:python3_host_skip_check=1
    else
        let g:python_host_prog = '/home/alex/.pyenv/versions/neovim2/bin/python'
        let g:python3_host_prog = '/home/alex/.pyenv/versions/neovim3/bin/python'
        let g:python3_host_skip_check=1
    endif
endif

" [davidhalter/jedi-vim]
let g:jedi#completions_enabled=0
let g:jedi#force_py_version=3
```

### Output of “:verbose JediDebugInfo”

```
#### Jedi-vim debug information
Using Python version: 2
 - global sys.version: `2.7.14 (default, Apr 18 2018, 01:36:32), [GCC 5.4.0 20160609]`
 - global site module: `/home/alex/.pyenv/versions/neovim2/lib/python2.7/site.pyc`
Jedi path: `/home/alex/dotfiles/nvim/plugged/jedi-vim/pythonx/jedi/jedi/__init__.pyc`
 - version: 0.12.0
 - environment: <Environment: 3.5.2 in /home/alex/.pyenv>
 - sys_path:
There was an error accessing jedi_vim.jedi: Vim(echo):E121: Undefined variable: u
Traceback (most recent call last):
  File "/home/alex/dotfiles/nvim/plugged/jedi-vim/pythonx/jedi_vim_debug.py", line 49, in display_debug_info
Error detected while processing function jedi#debug_info[28]..provider#python#Call:
line   18:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/alex/dotfiles/nvim/plugged/jedi-vim/pythonx/jedi_vim_debug.py", line 56, in display_debug_info
    echo("printf('%s', {0!r})".format(l))
  File "/home/alex/dotfiles/nvim/plugged/jedi-vim/pythonx/jedi_vim_debug.py", line 6, in echo
    vim.command('echo {0}'.format(msg))
  File "/home/alex/.pyenv/versions/neovim2/lib/python2.7/site-packages/neovim/api/nvim.py", line 272, in command
    return self.request('nvim_command', string, **kwargs)
  File "/home/alex/.pyenv/versions/neovim2/lib/python2.7/site-packages/neovim/api/nvim.py", line 170, in request
    res = self._session.request(name, *args, **kwargs)
  File "/home/alex/.pyenv/versions/neovim2/lib/python2.7/site-packages/neovim/msgpack_rpc/session.py", line 100, in request
    raise self.error_wrapper(err)
NvimError: Vim(echo):E15: Invalid expression: `%s`\', {0!r})".format(p))')
Error detected while processing function jedi#debug_info:
line   28:
E171: Missing :endif
```

**Comment by davidhalter** [comment]
It should not be a problem to use Python 2, because completions will still be in Python 3. As you can see in the debug output, it's still using a 3.5.2 environment. The debug function still seams to be buggy, but that's a separate issue.

Is this good enough for you?

**Comment by alwc** [comment]
Yup. It clarifies my doubts.

Initially I took a look at `:JediDebugInfo` because I got the same problem similar to [this issue (Exception: There's an issue (XXX != XXX) with the diff parser #1081)](https://github.com/davidhalter/jedi/issues/1081). Over there I've provided a reproducible case to get the same error.
